src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -296,6 +297,56 @@ public boolean isXmppConnected()
                 "return APP.conference._room.xmpp.connection.connected;"));
     }
 
+    /**
+     * {@inheritDoc}
+     * If it is not audio muted will be checking it for 5 seconds before
+     * returning a result.
+     */
+    @Override
+    public boolean isAudioMuted()
```

Comment:
If the implementation is waiting then for me it would make sense to change isAudioMuted to assertAudioMuted(boolean). Same for the video one.

"Participant2 is not audio muted" this text can be generated on assertFailure by the participant which then can use it's name.

Reply:
This is just checking whether it is audioMuted or not. The wait is just giving time the presence to be received. I just used the method in some asserts. 
This case is covered in the last commit of the PR, which reminds me that I changed the timeout there and I should revert it back to 5.
StartMutedTest for example, participant 1 is muted, participant 2 is joining but because it start muted and it is Safari, not dealing with video, we should skip waitForSendReceiveData cause there will be none.

Reply:
It feels weird that getter method is waiting for anything. It's implementation clearly looks like an assert.